source/server/hot_restart_impl.cc

```diff
@@ -29,7 +27,7 @@ namespace Server {
 
 // Increment this whenever there is a shared memory / RPC change that will prevent a hot restart
 // from working. Operations code can then cope with this and do a full restart.
-const uint64_t SharedMemory::VERSION = 10;
+const uint64_t SharedMemory::VERSION = 11;
```

Comment:
I think we should have a policy that if the version is incremented we should have a release note about it. Can you update the above comment to this effect and also add a release note?

Reply:
Certainly; done.